chalice/deployer.py

```diff
@@ -31,7 +31,8 @@
         "Sid": "",
         "Effect": "Allow",
         "Principal": {
-            "Service": "lambda.amazonaws.com"
+            "Service": "lambda.amazonaws.com",
+            "Service": "apigateway.amazonaws.com",
```

Comment:
@jamesls 
Thank you for the code review.

As you pointed out. Sorry. I'll fix it soon.